angularSTEP/src/app/recipe-page/recipe-page.component.ts

```diff
@@ -198,14 +196,14 @@ export class RecipePageComponent {
   }
 
   isRecipeInWishlist() {
-    if (this.user && this.loggedIn && this.id) {
+    if (this.user && this.signedIn && this.id) {
```

Comment:
Am I missing something or does `signedIn` not do anything more use than checking if `user` is truthy does?